docs/precommit.md

```diff
@@ -57,33 +57,33 @@ Git-format-staged requires Python v3 or v2.7. Python is usually pre-installed on
 <!--npm-->
 
 ```bash
-npx husky-init
+npx husky init
 npm install --save-dev git-format-staged
-npx husky set .husky/pre-commit "git-format-staged -f 'prettier --ignore-unknown --stdin --stdin-filepath \"{}\"' ."
+node --eval "fs.writeFileSync('.husky/pre-commit', 'git-format-staged -f \\'prettier --ignore-unknown --stdin --stdin-filepath \"{}\"\\' .\\n')"
```

Comment:
This command seems not excuteable. 

```
>node --eval "fs.writeFileSync('.husky/pre-commit', 'git-format-staged -f \\'prettier --ignore-unknown --stdin --stdin-filepath \"{}\"\\' .\\n')"
[eval]:1
fs.writeFileSync('.husky/pre-commit', 'git-format-staged -f \\'prettier --ignore-unknown --stdin --stdin-filepath "{}"\\' .\\n')
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^

SyntaxError: missing ) after argument list
```

replace `\\` with `\` seems works

Reply:
Hi, yes, it works, but isn't there a risk of compatibility problems with Windows if you just put a backslash?

Reply:
The error above was thrown when I ran on Windows (cmd.exe).